pkg/windows/windows.go

```diff
@@ -611,6 +611,9 @@ func (vm *windows) RestoreAWSRoutes() error {
 	if err != nil {
 		return err
 	}
+	if err := vm.ensureHNSNetworksAreRemoved(); err != nil {
```

Comment:
How do the HNS networks get re-created? Will the node need to be fully re-configured or will the periodic WICD reconciliation handle the re-creation?

Reply:
> Will the node need to be fully re-configured ?

No. Windows services will start as usual.